### ISSUE TITLE
SLM does not support two libraries on same drive

### ISSUE THREAD

**Issue by PatiB87** [RevoLand/Steam-Library-Manager#101]
I moved steam to a drive where a library already existed.
The steam folder includes an empty library itself
Therefore now in SLM the other library already present on the same drive but a different folder is not showing up anymore.
Yet, in the steam settings both are shown.

**Comment by RevoLand** [comment]
Hey, i've just released v1.7.0.0 which should fix the issues caused by the new Steam app. 

Please let me know if you encounter the issue again!

**Comment by PatiB87** [comment]
I don't think it did. :( On that drive it still only shows the empty "Steam" folder but not my "SteamLib" folder where all the games are installed actually.
